projects/angular/components/ui-suggest/src/ui-suggest.component.ts

```diff
@@ -1067,6 +1066,8 @@ export class UiSuggestComponent extends UiSuggestMatFormFieldDirective
         if (!refocus) {
             this._focusChanged(false);
         }
+
+        if (!this.expandInline) { this._suggestContainerOberserver.unobserve(this.suggestContainer.nativeElement); }
```

Comment:
I think the check should be like this
```suggestion
        if (!this.suggestContainer) { this._suggestContainerOberserver.unobserve(this.suggestContainer.nativeElement); }
```

Reply:
I removed this condition because `suggestContainer` will never be null because is the root element. I refactored the code and added the codition `!this.isExpandInline` because we need to do this just in this case